okd/src/microshift-okd-multi-build.Containerfile

```diff
@@ -39,6 +39,9 @@ COPY --chmod=755 ./okd/src/create_repos.sh ${REPO_CONFIG_SCRIPT}
 COPY --chmod=755 ./okd/src/configure.sh ${OKD_CONFIG_SCRIPT}
 COPY --from=builder /src/_output/rpmbuild/RPMS ${USHIFT_RPM_REPO_PATH}
 
+# Install nfv-openvswitch repo which provides openvswitch extra policy package
+RUN dnf install -y centos-release-nfv-openvswitch
```

Comment:
Can we add this to the RUN command below, not to create yet another layer?
Also, please explain in comment why this is necessary. 

P.S. In general, it's strange that the package deoendency is not pulled automatically. Is the dependency missing?

Reply:
> Can we add this to the RUN command below, not to create yet another layer? Also, please explain in comment why this is necessary.

I tried to add this in RUN command below, but that doesn't work because this package add the nfv-openshfit repo and once repo is added then only it able to pull packages from it.

> 
> P.S. In general, it's strange that the package deoendency is not pulled automatically. Is the dependency missing?

openvswitch package which comes from openshift-mirror-beta repo (https://mirror.openshift.com/pub/openshift-v4/dependencies/rpms/4.17-el9-beta/ ) which doesn't have it dependency `openvswitch-selinux-extra-policy` so either we need to update this mirror repo (not sure how) to add this dep package or use nfv repo which have required dependency.